micro_dl/preprocessing/preprocessing.md

```diff
@@ -0,0 +1,70 @@
+## Preprocessing output format
+
+The preprocessing step performs the following steps.
+
+* Segments the target images using selected segmentation algorithm in the configuration file(otsu, mem_detection, unimodal).
+
+* Stores the mask output as an extra channel in the zarr store with the name of the segmented channel with added subscript '_mask'. For instance, if the user segments the channel named 'Deconvolved-Nuc', the mask channel added will be called 'Deconvolved-Nuc_mask'. The datatype is the same as the channel which is segmented (most possibly float32).
+
+* Stores the information related to normalization of all input channels mentioned in the configuration. The dataset statistics are stored in the plate level .zattrs file, while the information specific to the position is added in the .zattrs file at each position. The details are explained below.
+
+Here is the structure of a 0.4 NGFF version zarr store wriiten using [iohub](https://github.com/czbiohub/iohub) for a dataset with a single condition and multiple imaging FOVs.
+
+```
+well0
```

Comment:
I'm not sure if it's a good idea to name this example `well0` since it's really a well but a row in NGFF.
Also it would be clearer to include the top level `.zarr` directory in the tree like [this](https://ngff.openmicroscopy.org/0.4/index.html#hcs-layout).

Reply:
Thank you @ziw-liu . This is a much clearer example. I have made the modification accordingly.